extensions/samples/msal-node-extensions/index.js

```diff
@@ -52,7 +52,9 @@ PersistenceCreator
         // get url to sign user in and consent to scopes needed for application
         pca.getAuthCodeUrl(authCodeUrlParameters).then((response) => {
             res.redirect(response);
-        }).catch((error) => console.log(JSON.stringify(error)));
+        }).catch((error) => {
+            console.error(error.errorMessage);
```

Comment:
This is out of scope for this PR. We can address this in a follow-up PR.

Reply:
+1 here, unless there's a reason the error needs to be swallowed it would be better to simply remove the log and let it throw. IMO this is an alternative approach to addressing the CVE so very much in scope.